definitions/json/AuthorisationCaseField.json

```diff
@@ -1,4 +1,34 @@
 [
```

Comment:
Large volume of changes in this file doesn't look right

Reply:
Surprised when I merged master also but fixed now